.github/workflows/test_release_docker_hub.yml

```diff
@@ -10,14 +10,14 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: ['3.8', '3.9', '3.10']
```

Comment:
We've mixed single and double quotes between different GH actions files. Not sure which is preferred, but it was making me twitch :smile: 
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```